pylint/extensions/_check_docs_utils.py

```diff
@@ -41,6 +41,21 @@ def returns_something(return_node):
     return not (isinstance(returns, astroid.Const) and returns.value is None)
 
 
+def inside_yields(node):
```

Comment:
I think this is the same as node.is_generator(), if the node is a FunctionDef node.